src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestClient.java

```diff
@@ -25,6 +25,20 @@ public interface SnowflakeStreamingIngestClient extends AutoCloseable {
    */
   SnowflakeStreamingIngestChannel openChannel(OpenChannelRequest request);
 
+  /**
+   * Drop the specified channel on the server using a {@link DropChannelRequest}
+   *
+   * <p>Note that {@link DropChannelRequest.DropChannelRequestBuilder#setClientSequencer(Long)}} can
+   * be used to drop a specific version of the channel and prevent accidentally dropping a channel
+   * concurrently opened by another client. If it is not specified, this call will blindly drop the
+   * latest version of the channel and any pending data will be lost. Also see {@link
+   * OpenChannelRequest.OpenChannelRequestBuilder#setDropOnClose(boolean)} to automatically drop
+   * channels on close.
+   *
+   * @param request the drop channel request
+   */
+  void dropChannel(DropChannelRequest request);
```

Comment:
Is this a public docstring?

Reply:
Yes, it is. Any suggestions to improve it?

Reply:
Good point. I have removed references to the sequencer from the public api now. The internal code still supports sequencer when invoked via `close` but that's not visible externally anymore.